### PR TITLE
Add more explanation for nsenter

### PR DESCRIPTION
**Pull request by wonderflow**
Now the README in nsenter is a little confused and hard for new commings to understand.

Signed-off-by: Sun Jianbo <wonderflow@zju.edu.cn>

### CODE REVIEW

**Comment by avagin** [comment]
LGTM, thanks

File: nsenter/README.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
 
-The `nsenter` package registers a special init constructor that is called before the Go runtime has 
-a chance to boot.  This provides us the ability to `setns` on existing namespaces and avoid the issues
-that the Go runtime has with multiple threads.  This constructor is only called if this package is 
-registered, imported, in your go application and the argv 0 is `nsenter`.
+The `nsenter` package registers a special init constructor that is called before 
+the Go runtime has a chance to boot.  This provides us the ability to `setns` on
+ existing namespaces and avoid the issues that the Go runtime has with multiple 
+ threads.  This constructor will be called if this package is registered, 
+ imported, in your go application.
```
Comment (hqhq) [inline nsenter/README.md]: redundant head spaces.

Comment (wonderflow) [inline nsenter/README.md]: Fixed .
Thanks

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**